project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
-addSbtPlugin("com.dwijnand"      % "sbt-dynver"   % "4.1.1")
-addSbtPlugin("org.foundweekends" % "sbt-bintray"  % "0.5.5")
-addSbtPlugin("de.heikoseeberger" % "sbt-header"   % "5.6.0")
-addSbtPlugin("org.scalameta"     % "sbt-scalafmt" % "2.4.6")
+addSbtPlugin("de.heikoseeberger" % "sbt-header"         % "5.6.0")
+addSbtPlugin("org.scalameta"     % "sbt-scalafmt"       % "2.4.6")
+addSbtPlugin("com.codecommit"    % "sbt-github-actions" % "0.14.2")
+addSbtPlugin("com.github.sbt"    % "sbt-ci-release"     % "1.5.10")
```

Comment:
Doesn't this come in transitively with sbt-github-actions?

Reply:
Solved.